snippets/assertions.json

```diff
@@ -0,0 +1,72 @@
+{
+	"tpass": {
+		"prefix": "tpass",
+		"body": "t.pass(${1:'${2:message}'});",
+		"description": "Passing assertion."
+	},
+	"tfail": {
+		"prefix": "tfail",
+		"body": "t.fail(${1:'${2:message}'});",
+		"description": "Failing assertion."
+	},
+	"tok": {
+		"prefix": "tok",
+		"body": "t.ok(${3:value}${1:, '${2:message}'});",
+		"description": "Assert that `value` is truthy."
+	},
+	"tnotOk": {
+		"prefix": "tnotOk",
+		"body": "t.notOk(${3:value}${1:, '${2:message}'});",
+		"description": "Assert that `value` is falsy."
+	},
+	"ttrue": {
+		"prefix": "ttrue",
+		"body": "t.true(${3:value}${1:, '${2:message}'});",
+		"description": "Assert that `value` is `true`."
+	},
+	"tfalse": {
+		"prefix": "tfalse",
+		"body": "t.false(${3:value}${1:, '${2:message}'});",
+		"description": "Assert that `value` is `false`."
+	},
+	"tis": {
+		"prefix": "tis",
+		"body": "t.is(${4:value}, ${3:expected}${1:, '${2:message}'});",
+		"description": "Assert that `value` is equal to `expected`."
+	},
+	"tnot": {
+		"prefix": "tnot",
+		"body": "t.not(${4:value}, ${3:expected}${1:, '${2:message}'});",
+		"description": "Assert that `value` is not equal to `expected`."
+	},
+	"tsame": {
+		"prefix": "tsame",
+		"body": "t.same(${4:value}, ${3:expected}${1:, '${2:message}'});",
+		"description": "Assert that `value` is is deep equal to `expected`."
+	},
+	"tnotSame": {
```

Comment:
> You can still write `tns` to get it, at least in Atom

Works in VS Code as well

> Maybe the id could be t.notSame() instead of tnotSame? And description could be prefixed with t.notSame() for clarity.

Works for me :)